src/fragments/start/getting-started/react/auth.mdx

```diff
@@ -32,16 +32,44 @@ Now that we have our authentication service deployed to AWS, it's time to add au
 
 Open __src/App.js__ and make the following changes:
 
-1. Import the `withAuthenticator` component:
+1. Import the `Authenticator` component:
 
 ```javascript
-import { withAuthenticator } from '@aws-amplify/ui-react'
+import { Authenticator } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
 ```
 
-2. Change the default export to be the `withAuthenticator` wrapping the main component:
+2. Wrap the `Authenticator` component around the contents of the return statement:
 
 ```javascript
-export default withAuthenticator(App)
+return (
+  <Authenticator>
+    <div style={styles.container}>
+      <h2>Amplify Todos</h2>
+      <input
+        onChange={event => setInput('name', event.target.value)}
+        style={styles.input}
+        value={formState.name}
+        placeholder="Name"
+      />
+      <input
+        onChange={event => setInput('description', event.target.value)}
+        style={styles.input}
+        value={formState.description}
+        placeholder="Description"
+      />
+      <button style={styles.button} onClick={addTodo}>Create Todo</button>
+      {
+        todos.map((todo, index) => (
+          <div key={todo.id ? todo.id : index} style={styles.todo}>
+            <p style={styles.todoName}>{todo.name}</p>
+            <p style={styles.todoDescription}>{todo.description}</p>
+          </div>
+        ))
+      }
+    </div>
+  </Authenticator>
+);
```

Comment:
I was testing this out, and I was receiving an error after logging in.
You have to wrap all children in a function, otherwise it will give you an error.

So something like this

```
 {({signOut}) => (
          <>
            <!-- Logged in Code goes here -->
           <button onClick={signOut}>Sign out</button>          
          </>
        )}
```

Reply:
Ahh good catch!